src/django/api/matching.py

```diff
@@ -228,6 +228,17 @@ def get_messy_items_for_training(mod_factor=5):
     return {str(i['id']): {k: clean(i[k]) for k in i if k != 'id'}
             for i in records}
 
+def load_gazetteer():
+            """
+            Load a preexisting dedupe.Gazetteer model by using the TrainedModel object in
+            the Django ORM.
+            """
+            retrieve_model_obj = TrainedModel.objects.get(is_active=True)
```

Comment:
The `get_active` manager method is literally this same line of code, but by using the abstraction we make sure that any future changes we make to the process of fetching the active model will automatically be used.

Since retrieve is a verb, `retrieve_model_obj`  looks like the name of a function to me. Consider changing this to something more noun-y.

```suggestion
            active_model = TrainedModel.objects.get_active()
```